### PR TITLE
reactivate rangorde for gemeenteraadsleden

### PR DESCRIPTION
**Pull request by Rahien**
## Description

Adds rangorde back for gemeenteraadsleden.

In prepare legislatuur sectie (rangorde is generated when generating rows):
image
In create mandataris form when selecting gemeenteraadslid:
image
In edit mandataris form when selecting gemeenteraadslid:
image
But not others (except schepenen):
image
Rangode is shown in gemeenteraadslid detail page
image

## How to test

- Generate some rows for gemeenteraadsleden in the prepare legislatuur section, see the rangorde appear
- notice that this does not happen for e.g. bcsd
- Add a gemeenteraadslid in prepare legislatuur, see you can add a rangorde
- edit a gemeenteraadslid in prepare legislatuur, see you can add a rangorde
- view the details of a gemeenteraadslid of previous governing periods, see you can view and edit rangorde

### CODE REVIEW

File: app/components/verkiezingen/prepare-legislatuur-section.js [hunk 1]
```diff
@@ -265,14 +268,15 @@ export default class PrepareLegislatuurSectionComponent extends Component {
     this.isGeneratingRows = false;
   }
 
-  onCreate = restartableTask(async ({ instanceId }) => {
+  @action
+  async onCreate({ instanceId }) {
     this.editMode = null;
     const mandataris = await this.store.findRecord('mandataris', instanceId);
```
Comment (Rahien) [inline app/components/verkiezingen/prepare-legislatuur-section.js:274]: weirdly, this function stopped after this promise (without exception) if it was a restartable task for me. changing it to action worked.

File: app/components/rdf-input-fields/mandataris-rangorde.hbs [hunk 2]
```diff
@@ -13,7 +13,7 @@
     @width="block"
     id={{this.inputId}}
     value={{this.rangorde}}
-    placeholder="Vul de rangorde in, bvb “eerste schepen”"
+    placeholder="Vul de rangorde in, bvb “Eerste schepen”"
```
[the remark this answers is not in the document]
Comment (Rahien) [reply app/components/rdf-input-fields/mandataris-rangorde.hbs:16]: yes i thought the same and didn't want to spend the time now to fix

File: app/utils/rangorde.js [hunk 3]
```diff
@@ -19,6 +19,86 @@ export const rangordeStringMapping = {
   Achtiende: 18,
   Negentiende: 19,
   Twintigste: 20,
+  Eenentwintigste: 21,
+  Tweeëntwintigste: 22,
+  Drieëntwintigste: 23,
+  Vierentwintigste: 24,
+  Vijfentwintigste: 25,
```
[the remark this answers is not in the document]
Comment (Rahien) [reply app/utils/rangorde.js:26]: copilot was such an amazing help here

File: app/components/mandatarissen/mandataris-card.hbs [hunk 4]
```diff
@@ -48,7 +48,7 @@
         {{/if}}
       </div>
     </div>
-    {{#if @mandataris.bekleedt.isSchepen}}
+    {{#if @mandataris.bekleedt.hasRangorde}}
```
[the remark this answers is not in the document]
Comment (Rahien) [reply app/components/mandatarissen/mandataris-card.hbs:51]: which is fine, as the gemeenteraadslid should have one, but i'll show /niet-beschikbaar/

[the remark this answers is not in the document]
Comment (JonasVanHoof) [reply app/components/mandatarissen/mandataris-card.hbs:51]: Niet beschikbaar

**Review by JonasVanHoof — APPROVED** [review]
capital letter and its great!

**Pull request merged**